synopsys-task/src/synopsys-task/input.ts

```diff
@@ -31,6 +31,20 @@ export function getInput(
   return "";
 }
 
+export function getArbitraryInputs(
+  yamlKey: string,
+  classicEditorKey: string,
```

Comment:
If this `classicEditorKey` is not used in this method then how the BlackDuck and Coverity inputs are getting used for Classic Editor? Is this working correctly for BlackDuck and Coverity?

Reply:
addressed